packages/ckeditor5-image/src/imageresize.js

```diff
@@ -124,28 +122,62 @@ export default class ImageResize extends Plugin {
  *			.then( ... )
  *			.catch( ... );
  *
- * If you want to have separate buttons for each option, pass their names instead:
+ * If you want to have separate buttons for each option, pass their names to the `image.toolbar` instead. Please keep in
```

Comment:
Which icons are available? How do I find them?

We should at least link to `module:image/imageresize/imageresizeui~ImageResizeOption`.